test/overloads.jl

```diff
@@ -160,3 +160,10 @@ using IfElse: ifelse
 @test isequal(Symbolics.derivative(signbit(x), x), 0)
 
 @test iszero(Num(0.0))
+
+
+# Tests for both norms
+x = Num.(rand(10))
+
+@test norm(x) == norm(Symbolics.value.(x))
```

Comment:
If this works, you could just use this as the implementation.

Reply:
Oh never mind, you're only testing with `rand(10)`. Should probably test with actual expressions though.

Reply:
Added a test with variables now.